src/bridge/test-peer.c

```diff
@@ -43,6 +45,17 @@ typedef CockpitChannelClass MockEchoChannelClass;
 
 G_DEFINE_TYPE (MockEchoChannel, mock_echo_channel, COCKPIT_TYPE_CHANNEL);
 
+static gboolean
+on_other_closed (CockpitTransport *transport,
+                 const gchar *problem,
+                 gpointer data)
+{
+  gboolean *flag = data;
+  g_assert (*flag == FALSE);
+  *flag = TRUE;
+  return FALSE;
+}
```

Comment:
This doesn't pass on or assert `problem`. AFAICS this is being called on an explicit "clean" close, so I guess not much doubt here (and this test is about a timeout, not a clean close), so fine.